### DISCUSSION TITLE
@yield and @section issue

### DISCUSSION THREAD

**Discussion by TitanDvd** [livewire/livewire#6173] [7. Bugs (archived)]
### Livewire version

V3

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

It seems to be a serious bug when you are using two or more @yield and @section.
1. Place two yields `@yield('content')` and `@yield('css')` in any layout. For example `components.some_layout.blade.php`
2. When rendering a component from layout above  use the `@section` statement for both `@yields` and set some content
3. You will notice that only one `@yield` is rendered

I use this method to extends a layout from component/render method
```
public function render()
    {
        return view('livewire.administration.departments-component')
                ->extends('components.some_layout')
                    ->section('content');
    }
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@TitanDvd thanks for reporting! A failing test PR #6135 has already been submitted that covers stacks, so I wonder whether yield and section are also related? I've added this discussion number to it, so it can be reviewed.

**Comment by TitanDvd** [comment]
Hi so far that was my only test on @yield and @section and I have to stop testing Lw V3 because @yield and @section problem makes me end all tests. I don't mind using Lw3 on production server, in fact Im currently developing online store with Lw2 but before it was with Lw3 but I have to stop using Lw3 because of that @yield @section problem. I will be waiting impatiently for an improved version that I can continue testing

**Reply by calebporzio** [reply]
its been fixed. The next beta release should take care of it
